catalog/main.go

```diff
@@ -7,6 +7,8 @@ import (
 	"log"
 	"os"
 	"path"
+	"path/filepath"
+	"strings"
 )
 
 const catalogFolder = "hub"
```

Comment:
Like in other tools, could we make this value configurable via cmd options? (optional and by default it will be `hub`).